core/shared/src/main/scala/fs2/hashing/HashAlgorithm.scala

```diff
@@ -27,8 +27,30 @@ sealed abstract class HashAlgorithm
 object HashAlgorithm {
   case object MD5 extends HashAlgorithm
   case object SHA1 extends HashAlgorithm
+  case object SHA224 extends HashAlgorithm
   case object SHA256 extends HashAlgorithm
   case object SHA384 extends HashAlgorithm
   case object SHA512 extends HashAlgorithm
+  case object SHA512_224 extends HashAlgorithm
+  case object SHA512_256 extends HashAlgorithm
+  case object SHA3_224 extends HashAlgorithm
+  case object SHA3_256 extends HashAlgorithm
+  case object SHA3_384 extends HashAlgorithm
+  case object SHA3_512 extends HashAlgorithm
   final case class Named(name: String) extends HashAlgorithm
```

Comment:
If `HashAlgorithm` is `sealed` then we technically can't add new `case`s in the future. There are various annoying workarounds, we could unseal it but make the ctor private for example ... or just ignore it and break compatibility in this relatively minor way.

Reply:
Thanks, I made it unsealed with private constructor.